Packages/com.unity.inputsystem/Documentation~/PreReleaseNotes.md

```diff
@@ -0,0 +1,94 @@
+
+## Project-wide actions and default actions
+
+Project-wide actions are similar to the actions you would previously define in an actions asset, however instead of being an asset that you create in the Editor, they are stored as part of your project’s settings, and are configured in the Project Settings window.
+
+Compared with the previous workflow of creating an Action asset, and setting up a reference to that asset to access in your code, project-wide actions reduce the number of steps to set up input in your project, and reduces complexity in your project.
+
+![The Input Actions settings panel in the Project Settings window, showing the default player actions.](images/ProjectSettingsInputActions.png)<br/>
+*The Input Actions settings panel in the Project Settings window, showing the default player actions.*
+
+The project-wide actions feature has some default action maps set up, which you can add to, modify or delete. They are actions which are useful in typical games, such as moving a player character with WSAD keys or a Joypad stick, pressing a button to jump or interact, as well as common UI controls such as pointing, submitting, or canceling within a user interface.
+
+**Note**: If you’d like to delete all the default actions so that you can start from an empty configuration, you don’t need to delete the actions individually. You can delete the default Action Maps, which deletes all the Actions contained in those maps in one go.
+
+### Reading project-wide actions
+
+You can access the project-wide actions in your script by using the [InputSystem.actions](../api/UnityEngine.InputSystem.actions.html) property. For example:
+
+	var myAction = InputSystem.actions.FindAction("Player/Jump");
+
+The above line of code reads the "Jump" action, from the “Player” action map, which is one of the default actions that comes with the new project-wide actions feature.
+
+Unlike Input Action assets, the project-wide actions are stored in your project’s Project Settings folder, so they do not appear in your Project window. The **InputSystem.actions** property is a built-in reference to that asset. This means you can use all the same techniques described throughout the rest of the documentation about [using action assets](Workflow-ActionsAsset), but instead of referencing an asset from your project, you can use the **InputSystem.actions property** in your scripts to reference the project-wide actions.
+
+For example, here is the script from the [Action Assets workflow page](Workflow-ActionsAsset), adapted to use the project-wide actions, and the default actions in the "Player" action map. 
+
+```
+using UnityEngine;
+using UnityEngine.InputSystem;
+public class ExampleScript : MonoBehaviour
+
```

Comment:
Redundant newline here - probably should be above the class line as:

using UnityEngine.InputSystem;

public class ExampleScript : MonoBehaviour
{